game-core/src/main/java/games/strategy/engine/random/RandomStatsDetails.java

```diff
@@ -68,7 +68,7 @@ private static DiceStatistic getDiceStatistic(final IntegerMap<Integer> stats, f
       double sumOfSquaredMeanDeviations = 0;
       // TODO: does this need to be updated to take data.getDiceSides() ?
       for (int i = 1; i <= diceSides; i++) {
-        sumOfSquaredMeanDeviations += (stats.getInt(i) - (localTotal / diceSides))
+        sumOfSquaredMeanDeviations += (double) (stats.getInt(i) - (localTotal / diceSides))
```

Comment:
@ssoloff Yes and no.
The code snippet you presented was one of them, but this is another occurence.

Reply:
That's the same thing, right?

```java
sumOfSquaredMeanDeviations += (double) (stats.getInt(i) - (localTotal / diceSides)) * (stats.getInt(i) - (localTotal / diceSides));
sumOfSquaredMeanDeviations += (double) (i) * (i);
sumOfSquaredMeanDeviations += ((double) i) * i;
```

That one might have been clearer by first extracting the mean deviation term and then summing the square:

```java
final double meanDeviation = stats.getInt(i) - (localTotal / diceSides);
sumOfSquaredMeanDeviations += meanDeviation * meanDeviation;
```

Reply:
@ssoloff Yes, my bad. I didn't realize that the multiplication was split over 2 lines, thus my confusion.
Thanks for clarifying. Makes sense now 👍